tests/rewards/test_reward_nets.py

```diff
@@ -43,10 +43,17 @@ def _potential(x):
     testing_reward_nets.make_ensemble,
 ]
 
+MakeForwardWrapper = Callable[[reward_nets.RewardNet], reward_nets.ForwardWrapper]
+MAKE_PREDICT_PROCESSED_WRAPPERS = [
+    lambda base: reward_nets.NormalizedRewardNet(base, networks.RunningNorm),
+]
```

Comment:
why is `MakeForwardWrapper` grouped with `MAKE_PREDICT_PROCESSED_WRAPPERS` and vice versa?

Reply:
:+1: